integration/apps/nekbone/build.sh

```diff
@@ -32,31 +32,23 @@
 
 set -x
 set -e
-set -x
 
 # Get helper functions
 source ../build_func.sh
 
-function get_nekbone_source {
-    dirname=$1
-    basedir=$PWD
-    tmpdir=$(mktemp -d)
-    cd $tmpdir
-    svn checkout https://repocafe.cels.anl.gov/repos/nekbone
-    mv nekbone/trunk/nekbone $basedir/$dirname
-    cd -
-    rm -rf $tmpdir
+get_nekbone_source() {
+    svn checkout https://repocafe.cels.anl.gov/repos/nekbone/trunk/nekbone ${1}
```

Comment:
The goal with the tmp directory was to get rid of the SVN data.  Be nice to keep that feature.

Reply:
Also, I've been trying to name $1 in the function with a local variable so it is clear what it is.